packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -33,56 +35,61 @@ const useIconStyles = makeStyles(theme => ({
     justifyItems: 'center',
     gridGap: 4,
     textAlign: 'center',
+    '&:active': {
+      cursor: 'grabbing',
```

Comment:
what's up with this? :) this is the cursor that's used for drag-and-drop, and why change the cursor at all?

Reply:
Ha, I was mainly testing your attention. But the feeling of this cursor is that you pressed (the pointed finger goes down) so the experience is good.